includes/class-wc-stripe-checkout-sessions-controller.php

```
@@ -33,88 +33,32 @@ public function create_checkout_session(): void {
-					],
-				],
+				'payment_intent_data'           => $this->build_payment_intent_data(),
 				'mode'                          => 'payment',
 				'adaptive_pricing'              => [
 					'enabled' => 'true',
 				],
 			];
 
+			if ( is_user_logged_in() ) {
+				try {
+					$stripe_customer = new WC_Stripe_Customer( WC()->customer->get_id() );
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '53', column '29'). (undefined)

(MissingImport)